src/search.c

```diff
@@ -70,7 +70,13 @@ int main( int argc, char *argv[] )
 		fprintf( stderr, "File not found\n" );
 		return( 1 );
 	}
-	printf( "%i usable mirrors:\n", search_getspeeds( res, i ) );
+	num_mirrors = search_getspeeds( res, i );
+	if ( j < 0 )
```

Comment:
this should have been substituted with num_mirrors ... will change and push again